core/storage/file/gae_models.py

```diff
@@ -83,27 +157,54 @@ class FileSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
 
 
 class FileSnapshotContentModel(base_models.BaseSnapshotContentModel):
-    """Class for storing the content of the file snapshots."""
+    """Class for storing the content of the file snapshots.
+
+    Attributes:
+        content: BlobProperty. The contents of a file snapshot.
+    """
 
     # Overwrite the superclass member to use a BlobProperty for raw strings.
     content = ndb.BlobProperty(indexed=False)
 
 
 class FileModel(base_models.VersionedModel):
-    """File data model, keyed by exploration id and absolute file name."""
+    """File data model, keyed by exploration id and absolute file name.
+
+    Attributes:
+        SNAPSHOT_METADATA_CLASS: FileSnapshotMetadataModel. The class which
+            stores the commit history of the file snapshots.
+        SNAPSHOT_CONTENT_CLASS: FileSnapshotContentModel. The class which
+            stores the content of the file snapshots.
+        content: BlobProperty. The contents of the file.
+    """
     SNAPSHOT_METADATA_CLASS = FileSnapshotMetadataModel
     SNAPSHOT_CONTENT_CLASS = FileSnapshotContentModel
 
     # The contents of the file.
     content = ndb.BlobProperty(indexed=False)
 
     def _reconstitute(self, snapshot_blob):
-        """Manually overwrite the superclass method."""
+        """Manually overwrite the superclass method. Reconstitutes a FileModel
```

Comment:
Probably should be: "Overrides the superclass method. Reconstitutes a FileModel instance from a snapshot." Similar changes below.

(You were right, "overwrite" is incorrect! Good catch.)